plugins/excel/core/src/main/java/org/pentaho/di/trans/steps/excelwriter/ExcelWriterStep.java

```diff
@@ -135,17 +134,17 @@ public boolean processRow( StepMetaInterface smi, StepDataInterface sdi ) throws
 
         // Let's remember where the fields are in the input row
         int outputFieldsCount = meta.getOutputFields().length;
-        data.commentauthorfieldnrs = new int[ outputFieldsCount ];
-        data.commentfieldnrs = new int[ outputFieldsCount ];
-        data.linkfieldnrs = new int[ outputFieldsCount ];
-        data.fieldnrs = new int[ outputFieldsCount ];
+        data.commentauthorfieldnrs = new int[outputFieldsCount];
```

Comment:
Please check your code style settings; these lines shouldn't have changed.

Reply:
code style needs spaces for array brackets?

Reply:
The array formatting should now be restored.